dsDisplay.c

```diff
@@ -444,7 +444,6 @@ dsError_t dsGetEDIDBytes(intptr_t handle, unsigned char *edid, int *length)
 {
 	dsError_t ret = dsERR_NONE;
 	uint8_t buffer[128];
```

Comment:
This shall not use 128; instead MAX_EDID_BYTES_LEN or such reference should be used.
Ref: https://github.com/rdkcentral/rdk-halif-device_settings/blob/bf57fa1d7aef6804dd110dd5141455987e2d65d3/include/dsDisplay.h#L209C79-L209C97